cpp-package/src/inc/data_filter.h

```diff
@@ -7,11 +7,31 @@
 #include "brainflow_exception.h"
 #include "data_handler.h"
 
+enum class LogLevels : int
```

Comment:
I think its not good to add it here... first of all there is such enum in BoardShim as I remember and more likely it will lead to multiple definitions error. It can be moved from both places to brainflow_constants.h or smth like that or to new header. Or you can include board_shim.h right here if needed

But I think that we can completely remove it from data_filter and ml_model and make set_log_level method private. So, in user API we will keep only enable\disable logger and in this case there is no need for such enum. (ok to hardcode 0, 2 and 6 internally)